thirdparty/mbedtls/LICENSE

```diff
@@ -200,3 +207,347 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+===============================================================================
+
+
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```

Comment:
For the LICENSE, I've actually be cutting the intro and GPLv2 sections to just keep Apache, "choosing" it as the license we're following.

I'm slightly worried that end users or license checking automation tools will freak out at the mention of GPL.

Reply:
Fixed :+1: